src/reynir_correct/pattern.py

```diff
@@ -902,11 +924,25 @@ def wrong_verb_use(
     def vera_að(self, match: SimpleTree) -> None:
         """ 'vera að' in front of verb is unneccessary """
         # TODO don't match verbs that allow 'vera að'
-        # TODO exclude sentences where the subject is not a noun (and later, where not alive)
         so = match.first_match("VP >> 'vera'")
         if so is None: return
         so = so.first_match("so")
         if so is None: return
+        subj = match.first_match("NP-SUBJ")
+        works = False
+        # TODO For now, only correct if the subject is a 1st or 2nd person pronoun or person name
+        allowed = ["PERSON"]
+        if not subj:
+            return
+        for x in subj.children:
+            if x.is_terminal and x.kind not in ["PUNCTUATION"]:
+                if x.kind in allowed:
+                    works = True
+                if x.cat and "pfn" in x.cat:
+                    if "p1" in x.all_variants or "p2" in x.all_variants:
```

Comment:
Neater (and probably performs better): `if {"p1", "p2"} & x_all_variants:`

Reply:
Done